test/relations.jl

```diff
@@ -287,152 +344,291 @@ end
         saturation_vapor_pressure(param_set, _T_triple, Ice()),
     ) ≈ 0.998 * q_tot / ρ_v_triple / ρ - 1
 
-    # energy functions and inverse (temperature)
+    # Helper function to test partial pressures
+    function test_partial_pressures(
+        T_test,
+        ρ,
+        q_partition,
+        test_name;
```

Comment:
Is `test_name` used anywhere?

Reply:
Good point. I added a @testset that echos the name for help in finding errors.